content/blog/2022/2022-02-14-spring-shell.md

```diff
@@ -0,0 +1,295 @@
+## Spring Shell
+[Spring Shell](https://spring.io/projects/spring-shell) allows us to build a command line application (Shell) using Spring's framework and all the advantages it provides.
+
```

Comment:
Please add the `{{% github %}}` shortcode here, linking to the example code in github (i.e. to the URL it will be after it has been merged). (see other articles files)

Reply:
Please add this code after the above paragraph: 

{{% github "https://github.com/thombergs/code-examples/tree/master/spring-boot/spring-shell" %}}